share/wake/lib/core/list.wake

```diff
@@ -38,15 +39,15 @@ package wake
 #   0, "1", Nil          # bad! This will trigger a type-check failure.
 # ```
 export data List a =
-# The empty list. Nil represents a list with no elements.
+    # The empty list. Nil represents a list with no elements.
     Nil
-# The prepend constructor. This is often called "cons".
-#
-# Examples:
-# ```
-#   10, list       # Add a number to `list`
-#   1, 2, 3, Nil   # Create a list of 3 elements
-# ```
+    # The prepend constructor. This is often called "cons".
+    #
+    # Examples:
+    # ```
+    #   10, list       # Add a number to `list`
+    #   1, 2, 3, Nil   # Create a list of 3 elements
+    # ```
```

Comment:
Cool. I like this. One of the first places I think the formatter IMPROVED the existing code instead of degrading it.